client/components/EventsList/index.jsx

```diff
@@ -46,6 +46,15 @@ export class EventsList extends React.Component {
         }
     }
 
+    previewOrEditEvent(event) {
+        // If we have the lock in this session, dispatch openPlanningEditor instead
```

Comment:
I would do this in the action in order to keep `onClick` and `onDoubleClick` simple without logic